### ISSUE TITLE
Soop_sst_trajectory Test: Missing char value encoder

### ISSUE THREAD

**Issue by bpasquer** [aodn/ncdfgenerator#68]
this is something you might have to look at @julian1 
Char is the encoder type of all the _quality_control variable ( BOM standard) in  soop_sst_trajectory  and this type of encoder is missing from the list of availaible  value encoder 
Got the following error when testing the ncdfgenerator:
"NcdfGeneratorException: Unrecognized tag 'char'
at au.org.emii.ncdfgenerator.NcdfDefinitionXMLParser$EncoderParser.parse"

(note that string value encoder seems to be missing too)

**Comment by julian1** [comment]
@bpasquer Yes, your right. There is support for byte (can also handle 1 byte strings) but not char. Should be simple to add, but probably needs to go into iteration planning.
